verification/v1/phoneVerification.yaml

```diff
@@ -52,6 +52,14 @@ paths:
           schema:
             type: string
             example: +491621234567
+        - name: countryIsoType
+          in: path
+          required: false
+          description: |
+            Option for country ISO code format. Defaults to iso3
+          schema:
+            type: string
```

Comment:
i would do the schema like that:

schema:
  type: string
  enum:
     - iso2
     - iso3
   example: iso2